Source/Scene/CreditDisplay.js

```diff
@@ -512,6 +513,14 @@ var defaultCredit;
 function getDefaultCredit() {
   if (!defined(defaultCredit)) {
     var logo = buildModuleUrl("Assets/Images/ion-credit.png");
+
+    // When hosting in a WebView, the base URL scheme is file:// or ms-appx-web://
```

Comment:
I know I'm wait late to the party here, but would setting CESIUM_BASE_URL to the http scheme be the better answer here and negate the need for this check?  Or is logo special and is the only thing that doesn't work with file://?

Reply:
In my project I host the Cesium files completely in the app, no files are referenced from the internet (except, of course, files from image providers). So setting a CESIUM_BASE_URL wouldn't fix that problem. The Web Worker files are loaded differently, I guess.

Reply:
Thanks for clarifying. It still feels like something strange is going on but I'm okay with the change in the meantime.